### ISSUE TITLE
Can't find your pre-trained models and trained models

### ISSUE THREAD

**Issue by TyroneLi** [facebookresearch/dmc-net#2]
Hi~I cannot access your trained models and pre-trained models by this link:
[https://github.com/facebookresearch/dmc-net/blob/master/dl.fbaipublicfiles.com/dmc-net/models.zip](url)
When I click this link, I get 404 error.Could you fix that and release your trained models and pre-trained models(such as _model_best.pth.tar).Thanks a lot~~
By the way could u provide more details about PWC-net for optical flow part inside your paper,if possibile, could your make this part code and models publicy?Best regards.

**Comment by stephenyan1231** [comment]
I believe the correct link should be: **dl.fbaipublicfiles.com/dmc-net/models.zip** .

**Comment by TyroneLi** [comment]
> I believe the correct link should be: **dl.fbaipublicfiles.com/dmc-net/models.zip** .

Yeap~I downloaded your models successfully through that link.Thank you!
But I am confused I use these command and hype-parameters to train dmc-gan but I can only get a little bit bigger than 29% cls accuracy training HMDB51:
`python train.py \
	--lr 0.01 \
	--lr-adv-g 1 \
	--lr-adv-d 0.01 \
	--lr-mse 10 \
	--lr_mse_mult 1 \
	--lr_d_mult 1 \
	--batch-size 30 \
	--arch resnet18 \
	--arch_estimator DenseNetTiny \
	--arch_d Discriminator3 \
	--data-name hmdb51 \
	--representation mv \
 	--data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos \
 	--flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 \
 	--train-list datalists/hmdb51_split1_train.txt \
 	--test-list datalists/hmdb51_split1_test.txt \
	--model-prefix HMDB/split1/DenseNetTiny+Discriminator3/ \
	--lr-steps 20 35 45 75 \
	--use_databn 0 \
	--epochs 100 \
	--epoch-thre 0 \
	--flow_ds_factor 0 \
	--gen_flow_or_delta 1 \
	--mv_minmaxnorm 1 \
	--no-accumulation \
	--gpus 6 7`
And I test your models it should be 47% cls accuracy.(for mv-gan-flow)
I also try using batch normalization layer to see if I could get better result:
`python train.py \
	--lr 0.01 \
	--lr-adv-g 1 \
	--lr-adv-d 0.01 \
	--lr-mse 10 \
	--lr_mse_mult 1 \
	--lr_d_mult 1 \
	--batch-size 30 \
	--arch resnet18 \
	--arch_estimator DenseNetTiny \
	--arch_d Discriminator3 \
	--data-name hmdb51 \
	--representation mv \
 	--data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos \
 	--flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 \
 	--train-list datalists/hmdb51_split1_train.txt \
 	--test-list datalists/hmdb51_split1_test.txt \
	--model-prefix HMDB/split1/DenseNetTiny+Discriminator3+BN/ \
	--lr-steps 20 35 45 75 \
	--use_databn 1 \
	--epochs 100 \
	--epoch-thre 0 \
	--flow_ds_factor 0 \
	--gen_flow_or_delta 1 \
	--mv_minmaxnorm 1 \
	--no-accumulation \
	--gpus 6 7`
But it does't help much.When I use Discriminator5 I could get 33% accuracy but it still much less than yours, 47%. I also try to adjust:
`--lr_mse_mult 1 \
--lr_d_mult 1 \`
to
`--lr_cls_mult 0.1 \
--lr_mse_mult 1 \
--lr_d_mult 0.1 \`
And get 38% accuracy but still almost 10 points less than 47%(your best model).
What should I adjust so I can get closer to your data accuracy?Thanls~

**Comment by TyroneLi** [comment]
> I believe the correct link should be: **dl.fbaipublicfiles.com/dmc-net/models.zip** .

I believe the correct link should be: dl.fbaipublicfiles.com/dmc-net/models.zip .
YeapI downloaded your models successfully through that link.Thank you!
But I am confused I use these command and hype-parameters to train dmc-gan but I can only get a little bit bigger than 29% cls accuracy training HMDB51:
python train.py --lr 0.01 --lr-adv-g 1 --lr-adv-d 0.01 --lr-mse 10 --lr_mse_mult 1 --lr_d_mult 1 --batch-size 30 --arch resnet18 --arch_estimator DenseNetTiny --arch_d Discriminator3 --data-name hmdb51 --representation mv --data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos --flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 --train-list datalists/hmdb51_split1_train.txt --test-list datalists/hmdb51_split1_test.txt --model-prefix HMDB/split1/DenseNetTiny+Discriminator3/ --lr-steps 20 35 45 75 --use_databn 0 --epochs 100 --epoch-thre 0 --flow_ds_factor 0 --gen_flow_or_delta 1 --mv_minmaxnorm 1 --no-accumulation --gpus 6 7
And I test your models it should be 47% cls accuracy.(for mv-gan-flow)
I also try using batch normalization layer to see if I could get better result:
python train.py --lr 0.01 --lr-adv-g 1 --lr-adv-d 0.01 --lr-mse 10 --lr_mse_mult 1 --lr_d_mult 1 --batch-size 30 --arch resnet18 --arch_estimator DenseNetTiny --arch_d Discriminator3 --data-name hmdb51 --representation mv --data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos --flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 --train-list datalists/hmdb51_split1_train.txt --test-list datalists/hmdb51_split1_test.txt --model-prefix HMDB/split1/DenseNetTiny+Discriminator3+BN/ --lr-steps 20 35 45 75 --use_databn 1 --epochs 100 --epoch-thre 0 --flow_ds_factor 0 --gen_flow_or_delta 1 --mv_minmaxnorm 1 --no-accumulation --gpus 6 7
But it does't help much.When I use Discriminator5 I could get 33% accuracy but it still much less than yours, 47%. I also try to adjust:
--lr_mse_mult 1 --lr_d_mult 1 
to
--lr_cls_mult 0.1 --lr_mse_mult 1 --lr_d_mult 0.1 
And get 38% accuracy but still almost 10 points less than 47%(your best model).
What should I adjust so I can get closer to your data accuracy?Thanls

**Comment by TyroneLi** [comment]
> I believe the correct link should be: **dl.fbaipublicfiles.com/dmc-net/models.zip** .

Yeap~I downloaded your models successfully through that link.Thank you!
But I am confused I use these command and hype-parameters to train dmc-gan but I can only get a little bit bigger than 29% cls accuracy training HMDB51:
`python train.py --lr 0.01 --lr-adv-g 1 --lr-adv-d 0.01 --lr-mse 10 --lr_mse_mult 1 --lr_d_mult 1 --batch-size 30 --arch resnet18 --arch_estimator DenseNetTiny --arch_d Discriminator3 --data-name hmdb51 --representation mv --data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos --flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 --train-list datalists/hmdb51_split1_train.txt --test-list datalists/hmdb51_split1_test.txt --model-prefix HMDB/split1/DenseNetTiny+Discriminator3/ --lr-steps 20 35 45 75 --use_databn 0 --epochs 100 --epoch-thre 0 --flow_ds_factor 0 --gen_flow_or_delta 1 --mv_minmaxnorm 1 --no-accumulation --gpus 6 7`
And I test your models it should be 47% cls accuracy.(for mv-gan-flow)
I also try using batch normalization layer to see if I could get better result:
`python train.py --lr 0.01 --lr-adv-g 1 --lr-adv-d 0.01 --lr-mse 10 --lr_mse_mult 1 --lr_d_mult 1 --batch-size 30 --arch resnet18 --arch_estimator DenseNetTiny --arch_d Discriminator3 --data-name hmdb51 --representation mv --data-root /data/lijinlong/datasets/Videos/HMDB/mpeg4_videos --flow-root /data/lijinlong/datasets/Videos/HMDB/new_optical_flow_fromMPEG4 --train-list datalists/hmdb51_split1_train.txt --test-list datalists/hmdb51_split1_test.txt --model-prefix HMDB/split1/DenseNetTiny+Discriminator3+BN/ --lr-steps 20 35 45 75 --use_databn 1 --epochs 100 --epoch-thre 0 --flow_ds_factor 0 --gen_flow_or_delta 1 --mv_minmaxnorm 1 --no-accumulation --gpus 6 7`
But it does't help much.When I use Discriminator5 I could get 33% accuracy but it still much less than yours, 47%. I also try to adjust:
`--lr_mse_mult 1 --lr_d_mult 1 `
to
`--lr_cls_mult 0.1 --lr_mse_mult 1 --lr_d_mult 0.1 `
And get 45% accuracy which is much better now.
Is that adjustment right or What should I adjust so I can get closer to your data accuracy?Thanls

**Comment by TyroneLi** [comment]
> I believe the correct link should be: **dl.fbaipublicfiles.com/dmc-net/models.zip** .

And could you provide more details about PMC-Net about Optical flow part because I think TV-L1 may cause some impact on optical flow extracted-floe-image quality,so I can not re-produce your accuracy successfully.

**Comment by zhengshou** [comment]
Thanks for pointing out the link issue. I will fix it. 

In our experiments, PWC-Net flow seems actually not as good as TV-L1 flow for supervision because PWC-Net flow is quite smooth, not as discriminative as TV-L1 yet discrimination is important for classification. You can refer to PWC-Net official repo for more details.

**Comment by Andrew-Zhu** [comment]
@TyroneLi  @zhengshou After I download the model.zip in dl.fbaipublicfiles.com/dmc-net/models.zip, I cannot unzip it. Could you tell me how you unzip the package? Thanks!
